editor/opening_hours_ui.hpp

```diff
@@ -0,0 +1,102 @@
+#pragma once
+
+#include "std/set.hpp"
+#include "std/vector.hpp"
+
+#include "3party/opening_hours/opening_hours.hpp"
+
+namespace editor
+{
+namespace ui
+{
+
+using TOpeningDays = set<osmoh::Weekday>;
```

Comment:
My point is that you can not check your code on all compilers and machines. If there's a room for undefined behaviour, in accordance with Merphie law something bad is going to happen, soon or later. Why not to prevent it right here and right now, with 100% guarantees? With only a 5-6 LOCS?